docs/modules/reference/pages/telemetryd/protocols/ipfix.adoc

```diff
@@ -75,7 +81,48 @@ The IPFIX TCP parser accepts packets received by a <<telemetryd/listeners/tcp.ad
 
 === Configuration and use
 
-This parser does not currently have any configurable parameters.
+.Optional parameters for the IPFIX TCP Parser
+[options="header" cols="1,3,1"]
+|===
+| Parameter
+| Description
+| Default
+
+| maxClockSkew
+| The maximum delta in seconds between exporter and Minion timestamps.
+| 0
+
+| clockSkewEventRate
+| Used to rate-limit clock skew events in seconds.
+| 3600
+
+| dnsLookupsEnabled
+| Used to enable or disable DNS resolution for flows.
+| true
+
+| sequenceNumberPatience
+| A value > 1 enables checking for sequence number completeness.
+The value gives the size of the history buffer allowing flows to be processed out of order.
```

Comment:
Is it the history buffer allowing flows to be processed out of order, or can someone use the value to allow flows to be processed out of order?

Reply:
The history buffer allows to process the flows out of order.